### DISCUSSION TITLE
Update the guidance around using adapter.js in WebRTC?

### DISCUSSION THREAD

**Discussion by wbamberg** [mdn/mdn-community#316] [Content] [answered]
(from Matrix chat, @SphinxKnight )

> I was browsing the WebRTC API landing page and saw one of the first section about adapter.js. When reading https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/adapter.js#what_adapter.js_does I'm not really sure it's useful to mention it (moreover invite people to use it at the very beginning). I don't use WebRTC but as of now, I'd be inclined to remove this: am I missing something?

**Comment by wbamberg** [comment]
We should certainly make some updates to this page:
- the page currently refers to the old  repo, https://github.com/webrtc/adapter, which is now a mirror of the current one which is at https://github.com/webrtcHacks/adapter.
- it seems that any compat issues are a bit different now (the page talks about prefixing, and different properties, while according to https://github.com/webrtcHacks/adapter the differences are mostly behavioral now)

It does seem like the adapter is still actively maintained though, so it probably is still relevant, but it would be good to check whether we should change the way we frame it, both at https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API#interoperability and in https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/adapter.js.

We could also consider retiring https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/adapter.js. and just have a section in https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API which links directly to the adapter. We could also consider not making this section the very first section of https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API.

Dominique suggested we ping the author of the adapter, @fippo , for their thoughts.

**Comment by fippo** [comment, marked as answer]
👋 adapter.js remains somewhat relevant (I am looking at you `connectionstatechange` event in Firefox as well as Safaris continued refusal to implement things it considers "legacy")
Retiring the standalone page (which is *really* outdated) and having a section at the bottom recommending its usage sounds like a good idea!

**Reply by wbamberg** [reply]
Thanks! I just filed https://github.com/mdn/content/pull/23528 for this. I'd be happy to hear your feedback on that PR if you have any :).
